src/cmd/build.js

```diff
@@ -104,7 +104,7 @@ async function defaultPackageCreator(
   let packageName = safeFileName(
     `${extensionName}-${manifestData.version}.zip`);
   let extensionPath = path.join(artifactsDir, packageName);
-  let stream = createWriteStream(extensionPath);
+  let stream = createWriteStream(extensionPath, {flags:'wx'});
```

Comment:
Oh, I see! Good point. I think adding the `wx` flags (in addition to the `fs.stat()` check) would be a good safety measure then.